MerlinAU.sh

```diff
@@ -3936,11 +3945,11 @@ Please manually update to version $minimum_supported_version or higher to use th
                 fi
 
                 # Convert version strings to comparable numbers
-                current_version="$(_ScriptVersionStrToNum_ "$BM_VERSION")"
-                required_version="$(_ScriptVersionStrToNum_ "1.5.3")"
+                local currentBM_version="$(_ScriptVersionStrToNum_ "$BM_VERSION")"
```

Comment:
This setting the value: current_version is wrong, that value is used for the firmware version elsewhere in the script and causes issues if we start populating it with the Backupmon version instead of the firmware version.

Reply:
![FB_IMG_1716348458645](https://github.com/ExtremeFiretop/MerlinAutoUpdate-Router/assets/1971404/5da9bd68-ef3a-453f-b357-1e1f955ee663)

Reply:
Probably more like "... a crime movie where you are the judge, jury & executioner." :>)